### ISSUE TITLE
making titanic examples generates warning

### ISSUE THREAD

**Issue by aoifecahill** [EducationalTestingService/skll#362]
```
conda_skll/lib/python3.4/site-packages/sklearn/utils/validation.py:395: DeprecationWarning: Passing 1d arrays as data is deprecated in 0.17 and will raise ValueError in 0.19. Reshape your data either using X.reshape(-1, 1) if your data has a single feature or X.reshape(1, -1) if it contains a single sample.
  DeprecationWarning)
```

**Comment by desilinguist** [comment]
Yeah I get a whole bunch more when I run the tests with scikit-learn
v0.19.0. I'll fix all of them together.

**Comment by aoifecahill** [comment]
👍

**Comment by desilinguist** [comment]
Addressed by #368.
